app/containers/Register/index.js

```diff
@@ -13,82 +13,179 @@ import {
     Button, 
     Text
 } from 'native-base';
-import { StyleSheet } from 'react-native';
+import { StyleSheet, Alert } from 'react-native';
+import ValidationComponent from 'react-native-form-validator';
+
 const role = [
-        { 
-            value: "key0",
-            label: "Attendee"
-        },
-        { 
-            value: "key1",
-            label: "Booth"
-        },
-        { 
-            value: "key2",
-            label: "Speaker"
-        },
-        { 
-            value: "key3",
-            label: "Admin"
-        },
-    ]
+    { 
+        value: "key0",
+        label: "Attendee"
+    },
+    { 
+        value: "key1",
+        label: "Booth"
+    },
+    { 
+        value: "key2",
+        label: "Speaker"
+    }
+];
 
-export default class Register extends Component {
+export default class Register extends ValidationComponent {
     constructor(props) {
         super(props);
         this.state = {
-            selected1: 'key0'
+            firstName: '',
+            LastName: '',
```

Comment:
Inconsistent naming, not match with the field name pass to `handleInputChange` function.